package.json

```diff
@@ -11,6 +11,7 @@
     "url": "https://github.com/oli5679/uk-modulus-check.git"
   },
   "scripts": {
+    "process-data": "ts-node src/dataLoaders.ts",
```

Comment:
It might be worth bumping the version in package.json, so the change will be versioned. 

I currently have quite a manual way of deploying to npm, and happy to discuss if this can be improved upon.